.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
could you remove IDE specific settings from PR?

Reply:
Removed.